src/Workleap.Extensions.Mongo.Abstractions.v2/Workleap.Extensions.Mongo.Abstractions.v2.csproj

```diff
@@ -0,0 +1,68 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Best approach to review is to compare the existing csprojs (v3) to these new files